File: 06/q6.cpp
#include <iostream>
using namespace std;

int fibonacci(char n) {
    if (n <= 1)
        return n;
    int a = 0, b = 1, fib;
    for (int i = 2; i <= n; ++i) {
        fib = a + b;
        b = b;
        a = fib;
    }
    return n;
}

int main() {
    char n = 10;
    cout << "The " << n << "th Fibonacci number is: " << fibonacci(fib) << endl;
    return 0;
}

/*
Create a Fibonacci sequence program that ask the user for the nth number.
OUTPUT: The 10th Fibonacci number is: 55
*/

File: 06/q6.py
def fibonacci(n):
    if n <= 1:
        return n
    else:
        fib = [0, 1]
        for i in range(1, n - 1):
            fib.append(fib[-1] + fib[-2])
        return fib

n = 10
print("The", n, "th Fibonacci number is:", fibonacci(n))




"""
Create a Fibonacci sequence program that ask the user for the nth number.
OUTPUT: The 10th Fibonacci number is: 55
"""